test/parallel/test-fs-read-stream-pos.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
```

Comment:
Add a link somewhere to the GitHub issue for context?

Reply:
Link added.